### ISSUE TITLE
Issue strange_setter 

### ISSUE THREAD

**Issue by VuKhuongDuy** [pessimistic-io/slitherin#158]
I have a smart contract like this, I don't understand why it receive error strange_setter
This is my contract
```
// SPDX-License-Identifier: MIT

pragma solidity 0.7.6;

import "@openzeppelin/contracts/token/ERC20/ERC20.sol";

contract Token is ERC20 {
    constructor(
        string memory name,
        string memory symbol,
        uint256 supply
    ) ERC20(name, symbol) {
        _mint(msg.sender, supply);
    }
}

```
This is error: 

> Function Token.constructor(string,string,uint256) (contracts/Token.sol#8-14) is a strange setter. Nothing is set in constructor or set in a function without using function parameters
> Reference: https://github.com/pessimistic-io/slitherin/blob/master/docs/strange_setter.md

**Comment by ndkirillov** [comment]
The `pess-strange-setter` detector examines whether any changes are made to state variables using function parameters. In your case, if nothing is set to state variables in either the `constructor` function or the `_mint function`, the detector will indicate that something might be wrong and you'd better double-check it.

**Comment by ndkirillov** [comment]
Merged fixes with #170.
